docs/api.rst

```diff
@@ -519,11 +552,11 @@ concatMap
 
   concatMap :: (a -> Stream b) -> Stream a -> Stream b
 
-Transform each event in stream into a stream, and then concatenate it onto the
-end of the resulting stream. Note that f must return a stream.
+Transform each event in ``stream`` into a new stream, and then concatenate each onto the
```

Comment:
To me, "new" implies that `f` must construct a stream to return, but any stream will do.  For example, it's valid to return the same `Stream b` for every `a`.  Granted, that's probably fairly rare, but it's allowed.

So, I think I prefer without "new".

Also, there's an opportunity to link to Stream:

```
Transform each event in ``stream`` into a :ref:`Stream`, and then ...
```

Overall, the docs aren't consistent in saying "stream" vs. "Stream" with link 😢 Laziness on my part, sorry!

Reply:
@briancavalier, I noticed that the lowercase stream is used when it refers to the literal block that follows.